app/src/main/java/com/doctor/yumyum/presentation/base/fragment/BaseFragment.kt

```diff
@@ -0,0 +1,33 @@
+package com.doctor.yumyum.presentation.base.fragment
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.Toast
+import androidx.annotation.LayoutRes
+import androidx.databinding.DataBindingUtil
+import androidx.databinding.ViewDataBinding
+import androidx.fragment.app.Fragment
+
+abstract class BaseFragment<T : ViewDataBinding>(@LayoutRes private val layoutResId: Int) :
+    Fragment() {
+    protected lateinit var binding: T
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        binding = DataBindingUtil.inflate(inflater, layoutResId, container, false)
+        return binding.root
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        binding.lifecycleOwner = this
```

Comment:
```suggestion
        binding.lifecycleOwner = viewLifecycleOwner
```
fragment는 lifecycleowner를 this로 받으면 안돼용~